.github/actions/run-embedding-sdk-storybook/action.yml

```diff
@@ -0,0 +1,33 @@
+name: Run Embedding SDK Storybook
```

Comment:
NIt: I would rename this workflow (both its name and file on disk) to `prepare-embedding-sdk-environment`.

1. we're not actually running tests in this action
2. we might find some solution other than the Storybook in the future